src/redisearch_rs/rlookup/src/row.rs

```diff
@@ -36,9 +36,9 @@ pub struct RLookupRow<'a, T: RSValueTrait> {
 }
 
 impl<'a, T: RSValueTrait> RLookupRow<'a, T> {
-    /// Creates a new `RLookupRow` with an empty [`RLookupRow::dyn_values`] vector and
-    /// a [`RLookupRow::sorting_vector`] of the given length.
-    pub fn new() -> Self {
+    /// Creates a new `RLookupRow` with an empty [`RLookupRow::dyn_values`] vector no
```

Comment:
Should say "with an empty [`RLookupRow::dyn_values`] vector **and** no [`RLookupRow::sorting_vector`]."?

Reply:
done!